lib/commands/certificate.js

```diff
@@ -276,12 +271,13 @@ commands.mobileInstallCertificate = async function mobileInstallCertificate (opt
 
   const tmpRoot = await tempDir.openDir();
   const tmpPort = await findAPortNotInUse(HOST_PORT_RANGE[0], HOST_PORT_RANGE[1]);
-  const tmpServer = new SubProcess('python3', ['-m', 'http.server', tmpPort], {
-    cwd: tmpRoot,
+  const configName = `appium.${CONFIG_EXTENSION}`;
+  const configPath = path.resolve(tmpRoot, configName);
+  const tmpServer = http.createServer(async function (_, res) {
```

Comment:
Thanks for getting rid of this Python dependency silliness.

Reply:
Silly me ;)